src/elements/content-sidebar/activity-feed/task-new/Assignees.js

```diff
@@ -53,33 +62,42 @@ class Assignees extends React.Component<Props> {
         const areThereMoreEntries = !!next_marker; // there are more assignees in another page of results
         const visibleAssignees = entries
             .slice(0, maxAvatars)
-            .map(({ id: assignmentId, target, status: assigneeStatus }) => {
+            .map(({ id, target, status, completed_at: completedAt }) => {
                 return (
-                    <Tooltip key={assignmentId} text={<AssigneeTooltipLabel user={target} status={assigneeStatus} />}>
+                    <Tooltip
+                        key={id}
+                        text={<AssigneeTooltipLabel user={target} status={status} completedAt={completedAt} />}
+                    >
                         <AssigneeStatus
-                            key={assignmentId}
-                            status={assigneeStatus}
+                            status={status}
                             user={target}
                             getAvatarUrl={getAvatarUrl}
                             data-testid="task-assignment-status"
                         />
                     </Tooltip>
                 );
             });
-        const allAssignees = entries.map(({ id: assignmentId, target, status: assigneeStatus }) => {
-            const statusMessage = statusMessages[assigneeStatus] || null;
+        const allAssignees = entries.map(({ id, target, status, completed_at: completedAt }) => {
+            const statusMessage = statusMessages[status] || null;
             return (
-                <li key={assignmentId} className="bcs-task-assignment-list-item">
+                <li key={id} className="bcs-task-assignment-list-item">
                     <AssigneeStatus
-                        status={assigneeStatus}
+                        status={status}
                         className="bcs-task-assignment-list-item-avatar"
                         user={target}
                         getAvatarUrl={getAvatarUrl}
                         data-testid="task-assignment-status"
                     />
                     <div className="bcs-task-assignment-list-item-details">
                         <div>{target.name}</div>
-                        <div className="bcs-task-assignment-list-item-status">{statusMessage}</div>
+                        {statusMessage && completedAt && (
```

Comment:
It looks like this is duplicated above with the exception of the class name. Should it be its own component and allow `className` to be passed in?

Reply:
Design keeps changing, sometimes they are the same string, sometimes not

Reply:
Maybe that's a bad excuse. I'll look into it.

Reply:
Thank you!